tests/test_default_reply_headers.js

```diff
@@ -112,14 +114,14 @@ test('default reply headers as functions work', async t => {
 
   const { headers } = await got('http://example.test')
 
-  t.deepEqual(headers, {
-    'content-length': message.length,
+  expect(headers).to.deep.equal({
+    'content-length': message.length.toString(),
```

Comment:
N.B. this test update does not reflect a change in Nock, but just using strict comparison now.